third_party/utf8_range/utf8_range.c

```diff
@@ -143,7 +137,7 @@ static size_t utf8_range_ValidateUTF8Naive(const char* data, const char* end,
   return err_pos + (1 - return_position);
 }
 
-#ifdef __SSE4_1__
+#if defined(__SSE4_1__) || defined(__ARM_NEON)
```

Comment:
I think `__aarch64__` is also required as 	vqtbl1q_u8 is only for A64 

https://developer.arm.com/architectures/instruction-sets/intrinsics/vqtbl1q_u8

Reply:
Good catch!

I try to use only macros defined by ACLE [1].
From the text below, the updated code checks both __ARM_NEON and __ARM_64BIT_STATE.
These two macros are defined since gcc-5.4 and clang-9.0. [2]
[1] https://arm-software.github.io/acle/main/acle.html#availability-of-advanced-simd-intrinsics
[2] https://godbolt.org/z/s918cP8fa

```
14.2.1 Availability of Advanced SIMD intrinsics
Advanced SIMD support is available if the __ARM_NEON macro is predefined (see Advanced SIMD architecture extension (Neon)). In order to access the Advanced SIMD intrinsics, it is necessary to include the <arm_neon.h> header.

  #if __ARM_NEON
  #include <arm_neon.h>
    /* Advanced SIMD intrinsics are now available to use.  */
  #endif
Some intrinsics are only available when compiling for the AArch64 execution state. This can be determined using the __ARM_64BIT_STATE predefined macro (see Instruction set architecture (A32/T32/A64)).
```